_data/payments.yml

```diff
@@ -127,6 +127,13 @@ websites:
       software: Yes
       doc: https://docs.payza.eu/docs/personal-manual-advanced-security#section-how-do-i-set-up-2-factor-authentication
 
+    - name: Plastiq
+      url: https://www.plastiq.com/
+      img: plastiq.png
+      tfa: Yes
+      sms: Yes
+      doc: https://help.plastiq.com/hc/en-us/articles/360006794893-How-do-I-enable-2-Step-Authentication-on-my-account-
```

Comment:
Could you change the doc url to `https://help.plastiq.com/hc/en-us/articles/360006794893`?